src/v2/models/AppState.js

```diff
@@ -141,8 +141,12 @@ export default Model.extend({
       // make sure to update the correct poll view's refresh value
       const currentFormName = this.get('currentFormName');
       const currentViewState = transformedResponse.remediations.filter(r => r.name === currentFormName)[0];
+      // Get new refresh interval for either: remediations, authenticator, or authenticator enrollment
+      const refreshInterval = currentViewState.refresh ||
```

Comment:
create a new function `getRefreshInterval` and move this logic there.. In case in future if we add more objects like `Authenticators`

Reply:
added